dbcon/mysql/idb_mysql.h

```diff
@@ -81,6 +81,7 @@ template <class T> bool isnan(T);
 #include "select_handler.h"
 #include "rpl_rli.h"
 #include "my_dbug.h"
+#include "sql_show.h"
```

Comment:
Why do you include this header?

Reply:
This is needed by the call to get_schema_tables_result() in create_columnstore_select_handler().